invenio_cli/cli/services.py

```diff
@@ -90,16 +90,19 @@ def status(cli_config, verbose):
     services = ["redis", cli_config.get_db_type(), "search"]
     statuses = commands.status(services=services, verbose=verbose)
 
-    messages = [
-        {"message": "{}: up and running.", "fg": "green"},
-        {"message": "{}: unable to connect or bad response.", "fg": "red"},
-        {"message": "{}: no healthcheck function defined.", "fg": "yellow"},
-    ]
-
-    for idx, status in enumerate(statuses):
-        message = messages[status]
+    messages = {
```

Comment:
minor/thinking out loud: these few lines of code are repeated in `containers.py`, and they are similar in objective to the `handle_process_response` function.

Can they be moved to a function? Maybe they are small enough to not need to, also _utils_ is not a good place to put things on (personal biased). So I'm not sure where to move it...

Reply:
Done, `containers` now invokes `setup` func `status`